sidechain/top-pool-rpc-author/Cargo.toml

```diff
@@ -15,6 +15,7 @@ std = [
     "itc-direct-rpc-server/std",
     "itc-tls-websocket-server/std",
     "itp-component-container/std",
+    "itp-enclave-metrics/std",
```

Comment:
Can you add `itp-ocall-api/std`?

Reply:
oh yes, forgot about that one, thanks misses detective 🕵️‍♀️